src/modules/85-cv/components/Notifications/NotificationsContainer.utils.ts

```diff
@@ -147,6 +144,18 @@ export function validateNotificationConditions(
     if (!condition?.condition && !values?.conditions?.[index]?.condition) {
       set(errors, `conditions.${index}.condition`, `${getString('cv.notifications.validations.conditionIsRequired')}`)
     }
+    if (condition?.condition?.value == 'CodeErrors' || values?.conditions?.[index]?.condition == 'CodeErrors') {
```

Comment:
can we use CodeErrors from enum 

```suggestion
    if (condition?.condition?.value == Condition.CodeErrors || values?.conditions?.[index]?.condition == 'CodeErrors') {
```